<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileFormatDialogBase</class>
 <widget class="QDialog" name="FileFormatDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Format</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>340</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Basic info.</string>
   </property>
   <widget class="QLineEdit" name="txt_Name">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>20</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_TimeInterval">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>80</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>23</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>52</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Control type : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>84</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_ControlType">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>50</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>340</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Storage condition</string>
   </property>
   <widget class="QLineEdit" name="txtIniStorage">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxStorage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxStorageRatio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>24</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial storage : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>56</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum storage : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>84</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum storage ratio : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>23</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>[m^3]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>55</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>[m^3]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>340</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Reservoir operation</string>
   </property>
   <widget class="QRadioButton" name="rdoAutomatic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic ROM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoRigid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rigid ROM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoUsingConstant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Using cosntant discharge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>84</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Constant discharge : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cms">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>81</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>86</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>[cms]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Constant_discharge">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>136</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>140</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>[cms] for </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_hours">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>136</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>140</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>[hours]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>340</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Read text file</string>
   </property>
   <widget class="QPushButton" name="btnFileFormat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>140</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>File format?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadFile">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>19</y>
      <width>140</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
